kubernetes/linux/setup.sh

```diff
@@ -29,6 +29,14 @@ rm -rf /usr/lib/ruby/gems/3.1.0/gems/openssl-3.0.1
 rm /usr/lib/ruby/gems/3.1.0/specifications/default/find-0.1.1.gemspec
 rm -rf /usr/lib/ruby/gems/3.1.0/gems/find-0.1.1
 
+# update the time and uri package to tackle the vulnerabilities in these gems
```

Comment:
@jatakiajanvi12 , I didnt see the change  for logging  Mariner version t in main.sh, will this be another PR?

Reply:
Yes let me add it in another PR...I havent tested that yet